src/status_im/contexts/wallet/bridge/input_amount/view.cljs

```diff
@@ -11,14 +11,16 @@
   (rn/use-unmount #(rf/dispatch [:wallet/clean-routes-calculation]))
   [rn/view {:style style/bridge-send-wrapper}
    [input-amount/view
-    {:current-screen-id :screen/wallet.bridge-input-amount
-     :button-one-label  (i18n/label :t/review-bridge)
-     :button-one-props  {:icon-left :i/bridge}
-     :on-confirm        (fn [amount]
-                          (rf/dispatch [:wallet/set-token-amount-to-bridge
-                                        {:amount   amount
-                                         :stack-id :screen/wallet.bridge-input-amount}]))
-     :on-navigate-back  (fn []
-                          (rf/dispatch [:wallet/clean-disabled-from-networks])
-                          (rf/dispatch [:wallet/clean-send-amount])
-                          (rf/dispatch [:navigate-back]))}]])
+    {:current-screen-id      :screen/wallet.bridge-input-amount
+     :button-one-label       (i18n/label :t/review-bridge)
+     :button-one-props       {:icon-left :i/bridge}
+     :enabled-from-chain-ids (rf/sub
```

Comment:
moved this to a prop of this component - for sending it's configurable and for bridging it's the other networks not being bridged to.